### ISSUE TITLE
Sections on contenteditable and designmode have insufficient discussion of keyboard accessibility

### ISSUE THREAD

**Issue by LJWatson** [w3c/editing#127]
Migrated from <a href="https://www.w3.org/Bugs/Public/show_bug.cgi?id=13739">Bugzilla #13739</a>

**Comment by gked** [comment]
Do you have anything specific in mind? Any known issues or lack of support when it comes to content editable and or design mode?

**Comment by carmacleod** [comment]
One area that is not well supported in contenteditable is using the keyboard to traverse a "word".
The user should be able to type `Ctrl + right arrow` to move the caret forward one word, and `Ctrl + left arrow` to move backward. But what is a "word"? Currently, the browser gets to decide. But it should be the author determining what makes the most sense for the app. For example, if the contenteditable is representing a code editor, then a "word" is not necessarily surrounded by the whitespace, comma, period of a typical "word" in a word processor. In fact, a code editor may want the definition of "word" to include strings consisting of only symbols. Maybe a "word" should only be a portion of aCamelCaseVariableName. Perhaps the definition of "word" is different in different languages. The point is that there is currently no API currently available to _page authors_ to let them specify "what is the 'word' at this caret position". The browsers have access to the platform API for this, but they currently don't pass it through.
